config/printer-elegoo-neptune3-pro-2023.cfg

```diff
@@ -0,0 +1,138 @@
+# Config for Elegoo Neptune 3 Pro (ZNP Robin Nano_DW V2.2)
+#
+# Based on:
+#
+# * https://github.com/NARUTOfzr/Neptune_3_Pro/blob/main/Source%20Code/PCB/ZNP%20Robin%20Nano_DW%20V2.2%20SCH.pdf
+# * https://github.com/NARUTOfzr/Neptune_3_Pro/issues/3#issuecomment-1364484284
+# * https://github.com/NARUTOfzr/Neptune_3_Pro/blob/main/Source%20Code/Marlin-2.1.1_TJC_1.1.5.1/Marlin/Configuration.h
+#
+# Enable advanced options, and set the baud rate of the firmware to 115200.
+#
+# `make flash` will not work for this board. Instead, do the following:
+#
+# * Rename `out/klipper.bin` to `out/ZNP_ROBIN_NANO.bin`
+# * Copy `out/ZNP_ROBIN_NANO.bin` to the root of an SD card formatted to FAT32
+# * Insert the card, then restart
+#
+# The display, if connected, will loop in the 'Updating firmware' screen. This
+# is normal, and the flash itself will only take a few seconds.
+#
+# See docs/Config_Reference.md for a description of parameters.
+
+# Core
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0
+restart_method: command
+baud: 115200
```

Comment:
Please do not select a custom baud other than the default 250000. Unless there is a very good reason to set this. Same goes for the comment in the build instructions

Reply:
There _is_ a very good reason. The board will suffer from random disconnects otherwise. I've tested this fairly extensively, and in fact, the Marlin config for this board requires this baud rate (which was what tipped me off).

Reply:
Then that is a good reason, though marlin baud rates work differently to klipper board connection, and 250000 default usually works. But as this is tested by you.

Thanks
James

Reply:
I'll note this in the comments at the top to clarify why I set it this way.